roles/rsyslog/vars/inputs/ovirt/main.yml

```diff
@@ -70,19 +77,27 @@ rsyslog_rulebase_ovirt_engine:
     filename: 'ovirt_engine.rulebase'
     nocomment: true
     sections:
-      - options: |-
-          version=2
-          rule=:%date:date-iso% %time:time-24hr%,%msec:number%%tz:word% %level:word%%-:whitespace%%class:string{"quoting.char.begin":"[","quoting.char.end":"]"}% %thread:string{"quoting.char.begin":"(","quoting.char.end":")"}% %correlationid:string{"quoting.char.begin":"[","quoting.char.end":"]"}% %message:rest%
+      - options: 'version=2
+
+        rule=:%date:date-iso% %time:time-24hr%,%msec:number%%tz:word%
+        %level:word%%-:whitespace%%class:string{"quoting.char.begin":\
+        "[","quoting.char.end":"]"}% %thread:string{"quoting.char.begin":\
+        "(","quoting.char.end":")"}%
```

Comment:
do you need a `\` at end of line here?

Reply:
Yes, otherwise a whitespace is added.

Reply:
I think this is a case where we want to ignore the line length checking because the long line is more readable than the wrapped line

or, move this into a file under files/